.github/issue_template/bug_report.md

```diff
@@ -0,0 +1,25 @@
+---
+name: Bug Report
+
+about: File a bug report.
+
+---
+
+ <!-- If you suspect this could be a bug, follow the template. -->
+
+### What version of Dgraph are you using? (result of `dgraph version`)
+
+
+### Tell us a little more about your go-environment? (result of `go version`)
+
+
+### Have you tried reproducing the issue with the latest release? 
+
+
+### What is the hardware spec (RAM, CPU, OS)?
+
+
+### Steps to reproduce the issue (command/config used to run Dgraph).
+
+
+### Expected behaviour and actual result.
```

Comment:
Do we want `en-gb` or `en-us`?  This is UK spelling of `behaviour`

Reply:
`en-us`, fixed it in https://github.com/dgraph-io/dgraph/pull/8159/commits/af3f0b0c7633c5c3676cff455264fa8cf2bd9e4c